src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -30,6 +31,8 @@ export interface OfferState {
   errorModalMessage: string
 }
 
+const logger = createLogger("Offer/index.tsx")
```

Comment:
#minor but maybe this should follow the same pattern as the path strings below (`"Order/Routes/Offer/index.tsx"`)

Reply:
Is it possible to to generate the parameter using `__filename`?